### PR TITLE
ThreeJS fiber experiments

### PR DESCRIPTION
**Pull request by hypevhs**
*Don't merge* unless you want to use this as the basis of the view and page style going forward.

Included some project-structure fixes that we can probably just cherry-pick later.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (nmoadev) [inline .vscode/launch.json:1]: 💯 I was wondering how to do this.

**Review by nmoadev — APPROVED** [review]
I'd be happy to merge as is. This is enough for me to start trying to do some plumbing: like trying to get game state down to a "Tile" component, etc.

@scottysseus I think it's just a matter of if this will cause you any merging conflict pain with your UI changes.

**Comment by scottysseus** [comment]
I think this looks good

I was thinking we could make the viewport occupy the full width and height, and place the summary and contract panes with absolute positions on top

Does that make sense and is that feasible?

**Comment by hypevhs** [comment]
@scottysseus Sounds good. R3F has an odd self-resizing feature that wasn't playing nice with the flexbox, so making the canvas fill the browser viewport might make the CSS neater.

**Pull request merged**